GTViewController/Classes/Custom Classes/GTFollowupModalView.m

```diff
@@ -70,16 +70,12 @@ - (instancetype)initFromElement:(TBXMLElement *)element withStyle:(GTPageStyle*)
         modalComponentElement = modalComponentElement->nextSibling;
     }
     
-    // there are too many elements, not sure what to do
-    if (totalUsedSpace >= totalVerticalSpace) {
-        
-    } else {
-        CGFloat availableBufferSpace = totalVerticalSpace - totalUsedSpace;
-        topLeadingSpace =       availableBufferSpace * 0.15;
-        betweenElementsSpace =  (availableBufferSpace * 0.2) / (numLabels + numButtonPairs + numTextInputs);
-    }
+    CGFloat availableBufferSpace = totalVerticalSpace - totalUsedSpace;
+    topLeadingSpace =       availableBufferSpace * 0.15;
+    betweenElementsSpace =  (availableBufferSpace * 0.2) / (numLabels + numButtonPairs + numTextInputs);
     
     int currentY = topLeadingSpace;
+    int inputFieldTag = 0;
```

Comment:
Sorry to do this to you but you are going to get tag conflicts with inputFieldTag starting at zero. I give all elements on the page tags. The way we are avoiding tag conflicts is to give each type of element a decimal digit. I think buttons get 0-9, panels get 10-90, etc. You will need to look through the GTPage class and find a range that is free and claim it.